Code/Samples/ShaderExplorer/ShaderExplorer.cpp

```diff
@@ -361,8 +394,13 @@ void ezShaderExplorerApp::AfterCoreSystemsStartup()
 
 void ezShaderExplorerApp::BeforeHighLevelSystemsShutdown()
 {
+  ezTelemetry::CloseConnection();
```

Comment:
why is it necessary to call CloseConnection here manually? this should automatically be taken care of elsewhere

does this function call it's base implementation (BeforeHighLevelSystemsShutdown()) ?

Reply:
So the `ezShaderExplorerApp` derives from the `ezApplication` class and not `ezGameApplicationBase` which would have called the function. The sample is pretty minimalistic which is great for porting stuff. Deriving from `ezGameApplication` would require solving many other problems first before you have a working app. I doubt Android is currently able to manage that.